oggm/tests/test_shop.py

```diff
@@ -402,6 +402,135 @@ def test_vdr(self, class_case_dir):
             # Fake tests, the plots look plausible
             np.testing.assert_allclose(d2.gradient.mean(), -0.0058, atol=.001)
             np.testing.assert_allclose(d2.temp_std.mean(), 3.35, atol=0.1)
+            
+   
+
+    def test_hydro_month_changes(self, hef_gdir):
+        # test for HEF if applying different hydro_months does the right thing
+        # check if mb of neighbouring hydro_months correlate
+        # do this for different climate scenarios 
+        
+        # maybe there is already somewhere an overview or a better way to get
+        # these dates, but I did not find it
+        base_data_time= {'CRU':{'start_year':1901,'end_year':2014},
+         'ERA5':{'start_year':1979,'end_year':2018},
+         'ERA5dr':{'start_year':1979,'end_year':2019},
+         'HISTALP':{'start_year':1850,'end_year':2014},
+          'CERA':{'start_year':1901,'end_year':2010},
+         'ERA5L':{'start_year':1981,'end_year':2018},
+        }
+        
+        gdir = hef_gdir
+        # this workkaround needed to get h,w because no fls there apparently
+        # maybe there is something easier in the testing environment
```

Comment:
https://github.com/OGGM/oggm/blob/4dfbfa39e8b22a13ecde936387ce3944ef7cb583/oggm/utils/_workflow.py#L2581

Reply:
changed, long story, I did not use get_inversion_flowline_hw, because there was no inversion_flowline when using the test environment with class_case_dir as in e.g. test_all_at-once, so I changed several things, among others I use instead the hef_gdir, ... still a bit confused about all these test things, but it will come ...